test/senderTest.js

```diff
@@ -30,7 +30,7 @@ describe('sensor SOC should exists', () => {
 
 describe('Sender data should exists', () => {
   it('should exists', () => {
-      expect(senderFetchReadings.fetchReadings).to.exist;
+      expect(senderFetchReadings.getSenderReadings).to.exist;

```

Comment:
Hi @ShainyAmin,
Adding @sudeeprp Sir as well as reviewer.

In case I missed (since I am new to JS), but shouldn't it be more like:
`expect(senderFetchReadings.getSenderReadings()).to.exist;`

When I run the `senderFetchReadings.getSenderReadings()`, I get output only as:
`[Function: getSenderReadings]`